commands/client_daemon_test.go

```diff
@@ -230,7 +231,12 @@ func TestPieceRejectionInProposeStorageDeal(t *testing.T) {
 func TestSelfDialStorageGoodError(t *testing.T) {
 	tf.IntegrationTest(t)
 
-	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fast.FilecoinOpts{})
+	// set block time sufficiently high that client can import its piece
```

Comment:
Do I understand correctly that this was introduced to reduce the chances of getting bitten by a race?  If so please document that a race (that we usually win) still exists.  Also consider referencing or creating an issue to remove this flaky behavior from daemon tests/this test.

Reply:
A storage client will wait a duration equal to [5 * `$BLOCKTIME`](https://github.com/filecoin-project/go-filecoin/blob/master/protocol/storage/client.go#L93) to complete the storage deal setup-stuff. This includes creating a payment, pinging the storage miner, etc.

If the environment is created without setting a block time explicitly, `$BLOCKTIME` [is equal to one millisecond](https://github.com/filecoin-project/go-filecoin/blob/5fbda21520cd282dbba60f0a9ddcc440e737e4f6/tools/fast/fastesting/basic.go#L66).

I don't think I'm masking over a race condition here. Rather that the default is too low for a storage client and miner to consummate a storage deal. Agree? Disagree?

Reply:
thx for explaining!

Reply:
no prob bob